integration_tests/tests/forseti/controls/scanner-external_project_access.rb

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+control "scanner - external project access" do
+  describe command("forseti inventory purge 0") do
+    its('exit_status') { should eq 0 }
+  end
+
+  describe command("forseti inventory create") do
```

Comment:
Lines 21-29 can be simplified. Instead of running `forseti inventory create`, `forseti inventory list`, and `forseti model create`, you can run `forseti inventory create --import_as {RANDOM}`. Fetch the "id" from this command the same way that is done on Line 25, and check exit status. Along with removing the inventory purge command, this will improve performance, make the test idempotent, and able to be run in parallel.

Reply:
the id is only needed to run the "forseti model create" command. If we are dropping this command then we don't need it right?

Reply:
Sorry, yes that's right we don't need the id. I didn't make it clear previously, when I mentioned {RANDOM} we need to actually provide a random string to make the model name unique. Then that random name needs to be provided to the `forseti model use` command that you have to replace the `model_new` name.